spacemacs/git-vcs.vim

```diff
@@ -1,27 +1,42 @@
 " Git clone
+let g:WhichKeyDesc_GitVcs_Clone = "<leader>gc clone"
 nnoremap <leader>gc    :action Git.Clone<CR>
 vnoremap <leader>gc    :action Git.Clone<CR>
 
+""" File Submenu ---------------------------------
+let g:WhichKeyDesc_GitVcs_File = "<leader>gf +file"
+
 " Show log of current file
+let g:WhichKeyDesc_GitVcs_File_Log = "<leader>gfl log"
 nnoremap <leader>gfl    :action Vcs.ShowTabbedFileHistory<CR>
 vnoremap <leader>gfl    :action Vcs.ShowTabbedFileHistory<CR>
+""" ---------------------------------------------
 
 " Git init
+let g:WhichKeyDesc_GitVcs_Init = "<leader>gi init"
 nnoremap <leader>gi    :action Git.Init<CR>
 vnoremap <leader>gi    :action Git.Init<CR>
 
 " Show local changes (git status)
+let g:WhichKeyDesc_GitVcs_Status = "<leader>gs status"
 nnoremap <leader>gs    :action Vcs.Show.Local.Changes<CR>
 vnoremap <leader>gs    :action Vcs.Show.Local.Changes<CR>
 
+""" VCS Submenu ---------------------------------
+let g:WhichKeyDesc_GitVcs_Vcs = "<leader>gv +vcs"
```

Comment:
I couldn't find the `gv` menu in my spacemacs.  I took a random guess at what the letter v stood for as `vcs`.  What does `v` actually stand for?

Reply:
You have to add this layer in order to see it: https://develop.spacemacs.org/layers/+source-control/version-control/README.html

Reply:
Thanks.  I added that layer, and see the 'v' menu now.  Will update the PR.